/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:24:06.246626 47961642525952 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:24:06.246626 47961642525952 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:06.246626 47961642525952 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:06.264352 47961642525952 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0419 14:24:06.264352 47961642525952 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:06.264352 47961642525952 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:06.273293 47961642525952 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:24:06.273293 47961642525952 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:06.273293 47961642525952 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:06.301079 47961642525952 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:24:06.301079 47961642525952 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:06.301079 47961642525952 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0419 14:24:12.979058 47961642525952 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0419 14:24:16.182640 47961642525952 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:24:16.182640 47961642525952 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.182640 47961642525952 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1517
V0419 14:24:16.182640 47961642525952 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:16.214637 47961642525952 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0419 14:24:16.214637 47961642525952 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.214637 47961642525952 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 1517
V0419 14:24:16.214637 47961642525952 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:16.226338 47961642525952 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:24:16.226338 47961642525952 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.226338 47961642525952 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 1517
V0419 14:24:16.226338 47961642525952 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:16.263583 47961642525952 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:24:16.263583 47961642525952 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.263583 47961642525952 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1517
V0419 14:24:16.263583 47961642525952 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:16.280590 47961642525952 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0419 14:24:16.280590 47961642525952 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.280590 47961642525952 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 796                                       
V0419 14:24:16.497714 47961642525952 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:24:16.497714 47961642525952 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:16.497714 47961642525952 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 1517
V0419 14:24:17.252342 47961642525952 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0419 14:24:17.252342 47961642525952 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.252342 47961642525952 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.276476 47961642525952 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0419 14:24:17.276476 47961642525952 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.276476 47961642525952 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.304622 47961642525952 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0419 14:24:17.304622 47961642525952 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.304622 47961642525952 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.320902 47961642525952 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0419 14:24:17.320902 47961642525952 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.320902 47961642525952 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.386706 47961642525952 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0419 14:24:17.386706 47961642525952 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.386706 47961642525952 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.642932 47961642525952 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0419 14:24:17.642932 47961642525952 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.642932 47961642525952 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.642932 47961642525952 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 796                                       
V0419 14:24:17.884359 47961642525952 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:24:17.884359 47961642525952 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:17.884359 47961642525952 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1517, actual 423
V0419 14:24:17.884359 47961642525952 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 3034
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:24:21.971866 47961642525952 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:24:21.971866 47961642525952 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:21.971866 47961642525952 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:21.971866 47961642525952 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:21.971866 47961642525952 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:22.020913 47961642525952 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0419 14:24:22.020913 47961642525952 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:22.020913 47961642525952 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:24:22.570068 47961642525952 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:22.570068 47961642525952 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:22.570068 47961642525952 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:22.675937 47961642525952 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:22.675937 47961642525952 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:22.675937 47961642525952 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:22.675937 47961642525952 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:25.884595 47961642525952 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:25.884595 47961642525952 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:25.884595 47961642525952 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:25.884595 47961642525952 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:25.884595 47961642525952 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200169.971121363                          
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:27.885494 47961642525952 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200171.972037558                          
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200169.971121363                          
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:27.955606 47961642525952 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200172.042208376                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200171.972037558                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200169.971121363                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:27.962750 47961642525952 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:28.026185 47961642525952 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:24:28.026185 47961642525952 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.026185 47961642525952 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.048057 47961642525952 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:24:28.048057 47961642525952 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.048057 47961642525952 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.067505 47961642525952 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:24:28.067505 47961642525952 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.067505 47961642525952 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.087188 47961642525952 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:24:28.087188 47961642525952 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.087188 47961642525952 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.104848 47961642525952 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:24:28.104848 47961642525952 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.104848 47961642525952 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.125145 47961642525952 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:24:28.125145 47961642525952 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.125145 47961642525952 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.170509 47961642525952 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:24:28.170509 47961642525952 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.170509 47961642525952 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.184085 47961642525952 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:24:28.184085 47961642525952 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.184085 47961642525952 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.201799 47961642525952 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:24:28.201799 47961642525952 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.201799 47961642525952 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.217897 47961642525952 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0419 14:24:28.217897 47961642525952 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.217897 47961642525952 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 3703                                       
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:28.486655 47961642525952 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:24:28.541424 47961642525952 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0419 14:24:28.541424 47961642525952 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.541424 47961642525952 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:24:28.578423 47961642525952 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0419 14:24:28.578423 47961642525952 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.578423 47961642525952 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:24:28.588629 47961642525952 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0419 14:24:28.588629 47961642525952 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.588629 47961642525952 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:24:28.600579 47961642525952 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0419 14:24:28.600579 47961642525952 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.600579 47961642525952 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:24:28.614321 47961642525952 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0419 14:24:28.614321 47961642525952 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.614321 47961642525952 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200172.049314338                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200172.042208376                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200171.972037558                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200169.971121363                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:28.658932 47961642525952 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200166.643980545                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200172.74549849                           
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200172.049314338                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200172.042208376                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200171.972037558                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200169.971121363                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200166.762589278                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200166.656726583                          
V0419 14:24:28.665479 47961642525952 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200166.643980545                          
W0419 14:24:28.665613 47961642525952 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0419 14:24:28.665613 47961642525952 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_10' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10)
W0419 14:24:28.665613 47961642525952 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 200166.643980545                          
W0419 14:24:28.665613 47961642525952 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0419 14:24:28.665613 47961642525952 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:29.624665 47961642525952 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0419 14:24:29.624665 47961642525952 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.624665 47961642525952 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:29.650273 47961642525952 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0419 14:24:29.650273 47961642525952 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.650273 47961642525952 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 797, actual 790
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:29.666099 47961642525952 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0419 14:24:29.666099 47961642525952 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.666099 47961642525952 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:29.681898 47961642525952 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0419 14:24:29.681898 47961642525952 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.681898 47961642525952 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:29.722682 47961642525952 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:24:29.722682 47961642525952 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.722682 47961642525952 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:29.722682 47961642525952 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 797                                         
V0419 14:24:29.722682 47961642525952 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 797                                         
V0419 14:24:29.754599 47961642525952 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:24:29.754599 47961642525952 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.754599 47961642525952 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:29.754599 47961642525952 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 797                                       
V0419 14:24:29.754599 47961642525952 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 797                                       
V0419 14:24:29.768286 47961642525952 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:29.768286 47961642525952 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:29.768286 47961642525952 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 797                                         
V0419 14:24:32.998944 47961642525952 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0419 14:24:32.998944 47961642525952 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:32.998944 47961642525952 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 797                                       
V0419 14:24:33.016538 47961642525952 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0419 14:24:33.016538 47961642525952 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:33.016538 47961642525952 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 797                                       
V0419 14:24:33.016538 47961642525952 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1517, actual 1536
V0419 14:24:33.268371 47961642525952 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:24:33.268371 47961642525952 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:33.268371 47961642525952 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:33.268371 47961642525952 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1517, actual 1536
V0419 14:24:33.268371 47961642525952 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1517, actual 1536
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1536
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 797                                         
V0419 14:24:33.509928 47961642525952 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1536
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1536
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 797                                       
V0419 14:24:33.548748 47961642525952 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1536
V0419 14:24:33.569410 47961642525952 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0419 14:24:33.569410 47961642525952 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:33.569410 47961642525952 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 3880, actual 3918
V0419 14:24:34.110964 47961642525952 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0419 14:24:34.110964 47961642525952 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.110964 47961642525952 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 797                                            
V0419 14:24:34.338846 47961642525952 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0419 14:24:34.338846 47961642525952 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.338846 47961642525952 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.356789 47961642525952 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0419 14:24:34.356789 47961642525952 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.356789 47961642525952 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.372965 47961642525952 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0419 14:24:34.372965 47961642525952 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.372965 47961642525952 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.632231 47961642525952 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0419 14:24:34.632231 47961642525952 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.632231 47961642525952 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.655509 47961642525952 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0419 14:24:34.655509 47961642525952 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.655509 47961642525952 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.672515 47961642525952 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0419 14:24:34.672515 47961642525952 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.672515 47961642525952 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.900612 47961642525952 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0419 14:24:34.900612 47961642525952 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.900612 47961642525952 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:34.982791 47961642525952 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:24:34.982791 47961642525952 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:34.982791 47961642525952 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:34.982791 47961642525952 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.001135 47961642525952 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:24:35.001135 47961642525952 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.001135 47961642525952 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.001135 47961642525952 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.019818 47961642525952 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:24:35.019818 47961642525952 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.019818 47961642525952 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.019818 47961642525952 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.039199 47961642525952 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:24:35.039199 47961642525952 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.039199 47961642525952 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.039199 47961642525952 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.056378 47961642525952 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:24:35.056378 47961642525952 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.056378 47961642525952 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.056378 47961642525952 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.075774 47961642525952 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:24:35.075774 47961642525952 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.075774 47961642525952 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.075774 47961642525952 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.120212 47961642525952 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:24:35.120212 47961642525952 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.120212 47961642525952 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.120212 47961642525952 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.133922 47961642525952 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:24:35.133922 47961642525952 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.133922 47961642525952 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.133922 47961642525952 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.151878 47961642525952 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:24:35.151878 47961642525952 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.151878 47961642525952 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0419 14:24:35.151878 47961642525952 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.168128 47961642525952 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0419 14:24:35.168128 47961642525952 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.168128 47961642525952 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 797                                        
V0419 14:24:35.168128 47961642525952 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 797                                        
V0419 14:24:35.337070 47961642525952 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0419 14:24:35.337070 47961642525952 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.337070 47961642525952 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.337070 47961642525952 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.378687 47961642525952 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0419 14:24:35.378687 47961642525952 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.378687 47961642525952 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.378687 47961642525952 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.390737 47961642525952 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0419 14:24:35.390737 47961642525952 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.390737 47961642525952 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.390737 47961642525952 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.403982 47961642525952 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0419 14:24:35.403982 47961642525952 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.403982 47961642525952 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.403982 47961642525952 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.421111 47961642525952 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0419 14:24:35.421111 47961642525952 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.421111 47961642525952 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.513506 47961642525952 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0419 14:24:35.513506 47961642525952 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.513506 47961642525952 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:35.584911 47961642525952 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0419 14:24:35.584911 47961642525952 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:35.584911 47961642525952 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:36.342483 47961642525952 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0419 14:24:36.342483 47961642525952 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:36.342483 47961642525952 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 797                                        
V0419 14:24:37.688833 47961642525952 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0419 14:24:37.688833 47961642525952 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:37.688833 47961642525952 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:37.954154 47961642525952 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0419 14:24:37.954154 47961642525952 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:37.954154 47961642525952 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.724145 47961642525952 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0419 14:24:38.724145 47961642525952 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.724145 47961642525952 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.836884 47961642525952 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:24:38.836884 47961642525952 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.836884 47961642525952 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.836884 47961642525952 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.836884 47961642525952 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.857728 47961642525952 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:24:38.857728 47961642525952 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.857728 47961642525952 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.857728 47961642525952 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.857728 47961642525952 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.878860 47961642525952 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:24:38.878860 47961642525952 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.878860 47961642525952 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.878860 47961642525952 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.878860 47961642525952 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.900220 47961642525952 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:24:38.900220 47961642525952 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.900220 47961642525952 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.900220 47961642525952 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.900220 47961642525952 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.919589 47961642525952 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:24:38.919589 47961642525952 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.919589 47961642525952 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.919589 47961642525952 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.919589 47961642525952 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.940894 47961642525952 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:24:38.940894 47961642525952 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.940894 47961642525952 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.940894 47961642525952 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.940894 47961642525952 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.989996 47961642525952 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:24:38.989996 47961642525952 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:38.989996 47961642525952 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:38.989996 47961642525952 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:38.989996 47961642525952 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:39.005535 47961642525952 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:24:39.005535 47961642525952 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:39.005535 47961642525952 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:39.005535 47961642525952 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:39.005535 47961642525952 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:39.024513 47961642525952 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:24:39.024513 47961642525952 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:39.024513 47961642525952 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:24:39.024513 47961642525952 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 797, actual 790
V0419 14:24:39.024513 47961642525952 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:39.200371 47961642525952 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:39.200371 47961642525952 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:39.200371 47961642525952 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:39.200371 47961642525952 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:42.595437 47961642525952 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:42.595437 47961642525952 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:42.595437 47961642525952 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:42.595437 47961642525952 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:42.595437 47961642525952 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 818                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:46.004695 47961642525952 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 815                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 818                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:49.554031 47961642525952 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 808                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 815                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 818                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:52.986008 47961642525952 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 806                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 808                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 815                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 818                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:56.502092 47961642525952 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 797                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 821                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 806                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 808                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 798                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 815                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 818                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 790                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:24:59.891380 47961642525952 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 797                                         
W0419 14:24:59.891608 47961642525952 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0419 14:24:59.891608 47961642525952 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0419 14:24:59.891608 47961642525952 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 797                                         
W0419 14:24:59.891608 47961642525952 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0419 14:24:59.891608 47961642525952 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/citeseer/SAGE/seed_1/sparsity_before.csv
                               layer_name  num_elements    sparsity
0  _orig_mod.encoder.graph_layer_1.weight      13712209   50.018097
1    _orig_mod.encoder.graph_layer_1.bias          3703  100.000000
2  _orig_mod.encoder.graph_layer_2.weight        473984   49.946201
3    _orig_mod.encoder.graph_layer_2.bias           128  100.000000
4      _orig_mod.encoder.decoder_1.weight      13712209   49.992222
5        _orig_mod.encoder.decoder_1.bias          3703   50.796651
6      _orig_mod.encoder.decoder_2.weight      13712209   50.003315
7        _orig_mod.encoder.decoder_2.bias          3703   50.094518
8         _orig_mod.encoder.linear.weight           768   46.744792
9           _orig_mod.encoder.linear.bias             6   33.333333
Saved sparsity snapshot to outputs_compiled/tran/citeseer/SAGE/seed_1/sparsity_after.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight      13712209  81.167702
1    _orig_mod.encoder.graph_layer_1.bias          3703  79.989198
2  _orig_mod.encoder.graph_layer_2.weight        473984  49.705686
3    _orig_mod.encoder.graph_layer_2.bias           128   4.687500
4      _orig_mod.encoder.decoder_1.weight      13712209  71.891451
5        _orig_mod.encoder.decoder_1.bias          3703  49.986497
6      _orig_mod.encoder.decoder_2.weight      13712209  85.829949
7        _orig_mod.encoder.decoder_2.bias          3703  49.905482
8         _orig_mod.encoder.linear.weight           768  55.338542
9           _orig_mod.encoder.linear.bias             6  33.333333
Saved forward timing report to outputs_compiled/tran/citeseer/SAGE/seed_1/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.002221       3005
1   _orig_mod.encoder.graph_layer_1             0.003008       3005
2   _orig_mod.encoder.graph_layer_2             0.002347       3005
3       _orig_mod.encoder.decoder_1             0.002616       3005
4       _orig_mod.encoder.decoder_2             0.002298       3005
5          _orig_mod.encoder.linear             0.000577       3005
6   _orig_mod.encoder.vq.project_in             0.002190       3005
7  _orig_mod.encoder.vq.project_out             0.003403       3005
8    _orig_mod.encoder.vq._codebook             0.003339       3005
CPU Memory Usage: 1285.14 MB
Max GPU Memory Used: 1364.04 MB
